trie/database.go

```diff
@@ -0,0 +1,12 @@
+package trie
+
+import (
+	"github.com/ChainSafe/gossamer/polkadb"
+	//"sync"
+)
+
+// Database is a wrapper around a polkadb
+type Database struct {
+	db polkadb.Database
+	//lock sync.RWMutex
```

Comment:
Can we remove the mutex or will we need it later?

Reply:
we will need it later when writing to the db